marlowe-dashboard-client/src/ContractHome/View.purs

```diff
@@ -77,10 +81,14 @@ contractCard index contractState =
 
     contractAcronym = contractTypeInitials metadata.contractType
 
-    stepNumber = currentStep contractState
+    stepNumber = currentStep contractState + 1
 
-    -- FIXME: hardcoded time slot
-    timeoutStr = "8hr 10m left"
+    mNextTimeout = contractState ^. _mNextTimeout
+
+    timeoutStr =
+      maybe "timed out"
+        (\nextTimeout -> humanizeDuration $ secondsDiff nextTimeout currentSlot)
+        mNextTimeout
```

Comment:
Now that this is counting down, it occurs to me we should speak to Russ about potentially showing it in a monospace font - otherwise it looks a bit jerky with the widths of the numbers changing all the time.

Reply:
👍🏻